### PR TITLE
changed documentation for Trainer.predict()

### PR DESCRIPTION
**Pull request by cbrew**
# Fix a documentation error in Trainer.predict

Fixes # (issue)

## Before submitting
- [*] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [*] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Review by sgugger — COMMENTED** [review]
Thanks for your PR, but your fix isn't correct either. The `predictions` could be a tuple with way more elements if the models return more than one logits, or if the user sets `output_attentions` to `True`. The actual type is impossible to give, so we should stay more evasive, stating it will depend on the number of values the model returns.
